datasets/iwslt2017/iwslt2017.py

```diff
@@ -0,0 +1,190 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace NLP Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IWSLT 2017 dataset """
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import nlp
+
+
+_CITATION = """\
+@inproceedings{cettoloEtAl:EAMT2012,
+Address = {Trento, Italy},
+Author = {Mauro Cettolo and Christian Girardi and Marcello Federico},
+Booktitle = {Proceedings of the 16$^{th}$ Conference of the European Association for Machine Translation (EAMT)},
+Date = {28-30},
+Month = {May},
+Pages = {261--268},
+Title = {WIT$^3$: Web Inventory of Transcribed and Translated Talks},
+Year = {2012}}
+"""
+
+_DESCRIPTION = """\
+The IWSLT 2017 Evaluation Campaign includes a multilingual TED Talks MT task. The languages involved are five:
+
+  German, English, Italian, Dutch, Romanian.
+
+For each language pair, training and development sets are available through the entry of the table below: by clicking, an archive will be downloaded which contains the sets and a README file. Numbers in the table refer to millions of units (untokenized words) of the target side of all parallel training sets.
+"""
+
+MULTI_URL = "https://wit3.fbk.eu/archive/2017-01-trnmted//texts/DeEnItNlRo/DeEnItNlRo/DeEnItNlRo-DeEnItNlRo.tgz"
+
+
+class IWSLT2017Config(nlp.BuilderConfig):
+    """ BuilderConfig for NewDataset"""
+
+    def __init__(self, pair, is_multilingual, **kwargs):
+        """
+
+        Args:
+            pair: the language pair to consider
+            is_multilingual: Is this pair in the multilingual dataset (download source is different)
+            **kwargs: keyword arguments forwarded to super.
+        """
+        self.pair = pair
+        self.is_multilingual = is_multilingual
+        super(IWSLT2017Config, self).__init__(**kwargs)
```

Comment:
```suggestion
        super().__init__(**kwargs)
```

We're Python 3 only so you can omit this